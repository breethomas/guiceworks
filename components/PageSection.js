// @flow
import React from 'react'
import css from '../styles/css'
import { maxWidthContent } from '../styles/jso'

const style = css(maxWidthContent)

export default (props: any) => (
  <section className={style} {...props} />
)
