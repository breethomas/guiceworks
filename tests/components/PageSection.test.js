import React from 'react'
import renderer from 'react-test-renderer'
import PageSection from '../../components/PageSection'

describe('PageSection', () => {
  let component

  beforeEach(() => {
    component = (
      <PageSection>
        PageSection
      </PageSection>
    )
  })

  it('shows the correct snapshot for the PageSection component tree', () => {
    const tree = renderer.create(component).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
