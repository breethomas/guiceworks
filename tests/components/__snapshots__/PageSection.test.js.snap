// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageSection shows the correct snapshot for the PageSection component tree 1`] = `
<section
  className={
    Object {
      "data-css-24spax": "",
    }
  }
>
  PageSection
</section>
`;
